greymatter/inputs.cue

```diff
@@ -1,62 +1,94 @@
 package greymatter
 
-config: {
-	// enable Spire-based mTLS
-	spire: bool | *false @tag(spire,type=bool)
-}
-
+// WARNING - Configurations in the mesh section must not be changed
+// arbitrarily because a tenant's mesh configurations hook into an existing
+// enterprise's core mesh configurations. Change these values in accordance
+// with enterprise-level configurations. This may require coordination with
+// an enterprise operations team.
 mesh: {
-	metadata: {
-		name: string | *"greymatter-mesh"
-	}
-	spec: {
-		zone: string | *"default-zone"
-		images: {
-			proxy: string | *"quay.io/greymatterio/gm-proxy:1.7.0"
-		}
+	// Mesh zone must match the enterprise zone so services are discovered
+	// by the greymatter control plane. Consult with the enterprise operations
+	// team if you are not sure about this value. It's important that the zone
+	// matches the enterprise zone before deploying tenant services.
+	zone: "default-zone"
```

Comment:
I think we wanted to rip this out right?

Reply:
Yes

Reply:
I'm waiting for direction on this btw and have pinged product about it.